drivers/watchdog/wdt_ite_it8xxx2.c

```diff
@@ -0,0 +1,273 @@
+/*
+ * Copyright (c) 2021 ITE Corporation. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT ite_it8xxx2_watchdog
+
+#include <logging/log.h>
+#include <drivers/watchdog.h>
+#include <soc.h>
+#include <errno.h>
+
+#define LOG_LEVEL CONFIG_WDT_LOG_LEVEL
+LOG_MODULE_REGISTER(wdt_ite_it8xxx2);
+
+#define IT8XXX2_WATCHDOG_MAGIC_WORD			0x5c
```

Comment:
This is a byte, not a word (neither 16- or 32-bit)

Reply:
Done